beatmap_collections/static/js/live_beatmap_card.js

```diff
@@ -1,6 +1,13 @@
 function RenderDemoCard(value) {
+    // If the value in form is blank, get an API call as the beatmap ID is 0
+    let api_url = '';
+    if (value === '') {
+        api_url = '/api/get_demo_card/0'
+    } else {
+        api_url = '/api/get_demo_card/' + value
+    }
```

Comment:
Actually, you can also write this 
```javascript
`/api/get_demo_card/${value || 0}`
```

Reply:
I will fix to your recommendation since it's look better